pkg/splunk/enterprise/util.go

```diff
@@ -913,16 +913,23 @@ func changePhaseInfo(desiredReplicas int32, appSrc string, appSrcDeployStatus ma
 				continue
 			}
 
+			// CSPL-1570: If this is the first time we are scaling up from replicas=1, then set the AuxPhaseInfo to Install complete.
```

Comment:
This is not an issue when scaling up from replicas > 1 because we will already have correct AuxPhaseInfo(with status as `AppPkgInstallComplete`) for both replicas the first time when afw scheduler runs. At the time of scaling from, let's say from 2 to 3, we will only update the AuxPhaseInfo for pod-2 and only install the app/s on that pod.